docs/TabbedForm.md

```diff
@@ -881,6 +880,59 @@ const ProductEdit = () => (
 
 Check [the RBAC `<TabbedForm>` component](./AuthRBAC.md#tabbedform) documentation for more details.
 
+### `<TabbedForm.Tab>`
```

Comment:
This chapter already contains a second level section with the same name, and I think it's confusing to repeat it here. I suggest removing this third-level section name. You will probably need to rephrase the following text, too. 

Same for TabbedShowLayout.